react/public/index.html

```diff
@@ -45,7 +45,20 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
+<<<<<<< HEAD
+    <!--
+          Notice the use of %PUBLIC_URL% in the tags above.
+          It will be replaced with the URL of the `public` folder during the build.
+          Only files inside the `public` folder can be referenced from the HTML.
+
+          Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+          work correctly both with client-side routing and a non-root public URL.
+          Learn how to configure a non-root public URL by running `npm run build`.
+        -->
+    <title>DiveVR</title>
+=======
     <title>Dive VR</title>
+>>>>>>> ad8fbc1ff779217b17e22a976b2ebf9af25ce3b0
```

Comment:
Can we fix this? It's causing the commit hash to show up in the login page:
![image](https://user-images.githubusercontent.com/36905285/69397183-1a106380-0c9a-11ea-9e68-5230fdba25bb.png)

Reply:
I don't know where this coming from, it starts happing after I pulled from the master.

Reply:
All you need to do is get rid of the extraneous code that doesn't belong there. For example, the `<<<<<<< HEAD`, `=======`, and `>>>>>>> ad8fbc1ff779217b17e22a976b2ebf9af25ce3b0`